README.md

```diff
@@ -347,13 +350,42 @@ jobs:
           asana-task-id: '${{ steps.find-asana-task-id.outputs.asanaTaskId }}'
 ```
 
+### Find multiple Asana task Ids in PR description
+Searches for Asana URLs in the PR description, given a prefix. Returns a comma-separated list of Asana Task Ids if found. The action will fail if no tasks are found.
+
+### `trigger-phrase`
+**Required** Prefix before the task i.e ASANA TASK: https://app.asana.com/1/2/3/.
```

Comment:
Should this also be optional to match the other action?

Reply:
Oops, yep, good catch! 👍